shippy-node-example/shippy-app/js/shippy.js

```diff
@@ -27,6 +28,14 @@ let Shippy = (function() {
 		isServing: null
 	};
 
+	const time = {
+		defaultWaitingTime: 15000,
```

Comment:
Is 15000 enough? If it takes longer than 15000ms for the first successor become the server, this will screw up the whole thing, right?

Reply:
Originally it was 21000ms. It was just taking too long to debug. But you're right, I'll leave it at 21000ms

Reply:
fixed